content/source/docs/enterprise/private/gcp-setup-guide.html.md

```diff
@@ -0,0 +1,353 @@
+| Recommended | 4-8 core | 16-32 GB RAM | 50GB    | Custom PostgreSQL Production |
+
+#### Hardware Sizing Considerations
+
+- The minimum size would be appropriate for most initial production
+  deployments, or for development/testing environments.
+
+- The recommended size is for production environments where there is a
+  consistent high workload in the form of concurrent terraform runs.
+
+### Object Storage (Cloud Storage)
+
+A [Regional Cloud Storage](https://cloud.google.com/storage/docs/storage-classes#regional) bucket must be
+specified during the PTFE installation for application data to be stored
+securely and redundantly away from the Compute Engine VMs running the PTFE
+application. This Cloud Storage bucket must be in the same region as the Compute Engine and Cloud SQL
+instances.
+Vault is used to encrypt all application data stored in the Cloud Storage bucket.  This
+still allows for the further [server-side
+encryption](https://cloud.google.com/storage/docs/encryption/)
+that Cloud Storage performs.
+
+### Vault Cluster
+
+In order to provide a fully stateless application instance, PTFE must be
+configured to speak with an [external Vault
+cluster](https://www.terraform.io/docs/enterprise/private/vault.html).  This
+reference architecture assumes that a highly available Vault cluster is
+accessible at an endpoint the PTFE servers can reach.
+
+### Other Considerations
+
+#### Additional GCP Resources
+
+In order to successfully provision this reference architecture you must
+also be permitted to create the following GCP resources:
+
+- Project
+- Network
+- Subnet
+- Firewall
+
+#### Network
+
+To deploy PTFE in GCP you will need to create new or use existing
+networking infrastructure. The below infrastructure diagram highlights
+some of the key components (network, subnets) and you will
+also have firewalls and gateway requirements. These
+elements are likely to be very unique to your environment and not
+something this Reference Architecture can specify in detail.
+
+#### DNS
+
+DNS can be configured external to GCP or using [Cloud DNS](https://cloud.google.com/dns/). The
+fully qualified domain name should resolve to the Alias IP using an A record.
+Creating the required DNS entry is outside the scope of this guide.
+
+#### SSL/TLS
+
+An SSL/TLS certificate is required for secure communication between
+clients and the PTFE application server. The certificate can be
+specified during the UI-based installation or the path to the
+certificate codified during an unattended installation.
+
+If a Classic or Application Load Balancer is used, SSL/TLS will be terminated there.
+In this configuration, the PTFE instances should still be configured to listen
+for incoming SSL/TLS connections but can do so using a self-signed certificate.
+
+Since the load balancer is configured to send traffic to port 443 on the instances
+as type `https`, will ignore the fact that it can't trust the self-signed certificate
+and operate correctly.
```

Comment:
Should this be "_we_ will ignore the fact"?

Reply:
I think Evan added this paragraph to a different cloud setup guide. I'll check with him on exactly what this should be.

Reply:
Updated to match Mary's new wording in the AWS guide